function-reference/navigateTo.md

```diff
@@ -0,0 +1,21 @@
+---
+description: >-
+  With navigateTo, we can navigate in between the internal pages of appsmith. It could be triggered on any widget action like Button onClick, Dropdown onOptionChange etc. Create a new Page with a valid page name before invoking this function.
+---
+
+# navigateTo 
+
+## Signature
+
+```text
+navigateTo(pageName: string) -> void
+```
+
+#### Arguments
+
+| **Argument Name** | **Description** |
+| :--- | :--- |
+| **pageName** |  Page name to which we would like to be transported. It's case sensitive. |
```

Comment:
@visibleajay the query params argument is missing here, can you update the table and the gif to reflect that

Reply:
Only one box is provided to provide the variable. Any hints on providing and accessing the second parameter.
<a href="https://ibb.co/X3xd4L7"><img src="https://i.ibb.co/VQYhN9g/one-box.png" alt="one-box" border="0"></a>